_stylesheets/03-molecules/_search.scss

```diff
@@ -1,81 +1,88 @@
-//search off
-.search-section {
-  background: rgba(68, 0, 39, .98);
-  height: 0;
-  position: fixed;
-  top: 0;
-  left: 0;
-  bottom: 0;
-  z-index: 9999;
-  overflow-y: scroll;
-  width: 100%;
-  transition-duration: 0.5s;
-  transition-timing-function: cubic-bezier(0, 1, 0.5, 1);
-
-  a {
-    color: white;
-    &:hover {
-      color: white;
-      text-decoration: underline;
+.search {
+  &__container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+
+    @include breakpoint($bp-small-landscape) {
+      flex-wrap: nowrap;
     }
   }
 
-  article {
-    padding-bottom: 10px;
+  &__submit {
+    margin-top: $base-margin;
+
+    @include breakpoint($bp-small-landscape) {
+      margin-top: 0;
+      margin-left: $base-margin;
+    }
   }
-}
 
-//search box overlay over the whole site
-.search-section.on {
-  opacity: 1;
-  height: 100vh;
-  transition-duration: 0.5s;
-  transition-timing-function: ease-in;
-}
+  &__input {
+    flex-basis: 75%;
+  }
 
-//content wrapper and X icon
-.search-section .content-wrapper {
-  max-width: 940px;
-  margin: 100px auto 50px auto;
-  text-align: left;
-  position: relative;
-  padding: 0 20px;
-
-  .close-out {
-    position: absolute;
-    right: 20px;
-    top: -45px;
+  &__results {
+    background: $probo-pale;
+    margin: $base-margin * 2 0;
   }
-}
 
-.search-section .fa-times {
-  font-size: 30px;
-}
+  &__results-count {
+    padding: $base-margin;
+  }
+
+  &__result {
+    padding: $base-margin * 2 0;
+    margin: 0 $base-margin;
+    border-bottom: 1px solid $probo-gray-light;
+    word-wrap: break-word;
+
+    &:last-of-type {
+      border-bottom: none;
+    }
+  }
+
+  &__result-title {
+    margin: 0;
+  }
 
-.search-section {
-  input[type=text] {
-    width: 100%;
-    font-family: fontawesome, arial;
-    height: 60px;
-    line-height: 60px;
+  &__result-link {
+    font-family: sans-serif;
+    color: $probo-purple-dark;
```

Comment:
Make this smaller, since it is really just extra, contextual content. Maybe equivalent to 12px.